domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/FullDefenceResponse.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.cmc.domain.models;
```

Comment:
doesn't there need to be a database migration for this to work?

Reply:
Yeah, I realised this just after creating this PR. Will add DB migration and update it.

Reply:
If possible the DB migration should come in a separate PR with the frontend being already updated to send the data.

Our new environment that we are working towards requires this and we should start getting into the practice of it.

Alternatively you can not do this and we can do it the harder way when we need to, up to you?

Reply:
@timja-kainos  I have added DB migration, please have a look with your experienced eyes.